Spring-Project/src/main/java/com/epam/project/spring/repository/impl/ProductRepositoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.project.spring.repository.impl;
+
+import com.epam.project.spring.model.Product;
+import com.epam.project.spring.repository.ProductRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ProductRepositoryImpl implements ProductRepository {
+
+    private final List<Product> listProduct = new ArrayList<>();
+    private static int idCount = 1;
+
+    @Override
+    public Product getProduct(Integer id) {
+        return listProduct.stream().filter(product -> product.getId().equals(id)).findFirst().orElseThrow(() -> new RuntimeException("Product not found"));
```

Comment:
Hi Zenyk. Please configure _chained method_ code style in the IntelliJ IDE in the following way:
1. Please open Preferences -> Code Style -> Java
2. Set **Chained method calls**: _Wrap always_
3. Please save changes and press Ctrl - Alt - L. Apply changes in all required places